api/csi.go

```diff
@@ -350,6 +350,8 @@ type CSIVolume struct {
 
 	CreateIndex uint64
 	ModifyIndex uint64
+	CreateTime  int64
+	ModifyTime  int64
```

Comment:
Similar to @schmichael's comment for the structs package, because this is the public API it'd be especially nice for us to document that these are unix nano.